cmd/jujud/agent/agentconf/export_test.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package agentconf
+
+import (
+	"github.com/juju/juju/agent"
+)
+
+func NewAgentConfForTest(dataDir string, cfg agent.ConfigSetterWriter) AgentConf {
```

Comment:
We shouldn't be creating new _export_test.go_ files.
This should be in _package_test.go_, preferable as a method on a base suite.